src/libraries/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/PerformanceData/CounterSet.cs

```diff
@@ -195,7 +195,7 @@ public CounterSetInstance CreateCounterSetInstance(string instanceName)
                     {
                         if (_provider == null)
                         {
-                            throw new ArgumentException(SR.Format(SR.Perflib_Argument_ProviderNotFound, _providerGuid), "ProviderGuid");
+                            throw new ArgumentException(SR.Format(SR.Perflib_Argument_ProviderNotFound, _providerGuid), nameof(instanceName));
```

Comment:
Is this correct?  We're throwing because _provider was null; how does that relate to `instanceName`?

Reply:
I changed it because message already saying that provider not found and passing the `_providerGuid` value and there is no argument "ProviderGuid", but of course its not what we want i will suppress

Reply:
Maybe better to change with:
`throw new InvalidOperationException(SR.Format(SR.Perflib_InvalidOperation_NoActiveProvider, _providerGuid));` this is what it throws when it is null in row 188, not sure why its checking for null again, but seems because of multithreading, Doc saying 
![image](https://user-images.githubusercontent.com/45579687/81208622-34a67280-8f84-11ea-8641-08712bd0f555.png)